src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1596,6 +1600,7 @@ static void _copyAttr(SvgNode* to, const SvgNode* from)
     }
     //Copy style attribute;
     memcpy(to->style, from->style, sizeof(SvgStyleProperty));
+    if (from->style->fill.paint.url) to->style->fill.paint.url = new string(from->style->fill.paint.url->c_str());
```

Comment:
I think, a case for stroke should also be added.
```suggestion
    if (from->style->fill.paint.url) to->style->fill.paint.url = new string(from->style->fill.paint.url->c_str());
    if (from->style->stroke.paint.url) to->style->stroke.paint.url = new string(from->style->stroke.paint.url->c_str());
```

Reply:
good point... I missed it

Reply:
updated.